openhands/runtime/impl/remote/remote_runtime.py

```diff
@@ -341,8 +341,6 @@ def _wait_until_alive_impl(self):
                 f'Runtime (ID={self.runtime_id}) is not yet ready. Status: {pod_status}'
             )
         elif pod_status in ('failed', 'unknown', 'crashloopbackoff'):
-            # clean up the runtime
-            self.close()
```

Comment:
CC @xingyaoww not sure if this was intentional in eval scenarios, but we think this is what's causing a lot of the pain in prod

Reply:
Let's remove it then! I think this is added early on, and now we can proper handling of AgentRuntime* exception in the eval loop and will perform runtime.close() on the outside - so this should be safe to remove